Assets/MixedRealityToolkit/_Core/Inspectors/MixedRealityManagerInspector.cs

```diff
@@ -93,6 +93,7 @@ public override void OnInspectorGUI()
         public static void CreateMixedRealityManagerObject()
         {
             Selection.activeObject = MixedRealityManager.Instance;
+            var playspace = MixedRealityManager.Instance.MixedRealityPlayspace;
```

Comment:
```suggestion
            var playspace = MixedRealityToolkit.Instance.MixedRealityPlayspace;
```

Reply:
Yeah, thanks for that. Had to rename it to match how it was, now I got to rename it back to how I originally copied it :P